manticore/core/state.py

```diff
@@ -415,6 +434,13 @@ def concretize(self, symbolic, policy, maxcount=7):
             else:
                 # We assume the path constraint was feasible to begin with
                 vals = (True,)
+        elif policy == "EXPLICIT":
+            if explicit_values:
+                for val in explicit_values:
```

Comment:
Do we want to warn or `assert` here if `explicit_values` is `None` or an empty list?

Reply:
Never mind, we'll just return an empty list if the above is true.